packages/client/src/write.ts

```diff
@@ -18,7 +19,9 @@ import {
 import { normalize, packetToBytes } from 'viem/ens'
 import { privateKeyToAccount } from 'viem/accounts'
 
-import { abi as l1Abi } from '@blockful/contracts/out/L1Resolver.sol/L1Resolver.json'
+import { abi } from '@blockful/contracts/out/DatabaseResolver.sol/DatabaseResolver.json'
```

Comment:
```suggestion
import { abi as databaseResolverAbi } from '@blockful/contracts/out/DatabaseResolver.sol/DatabaseResolver.json'
```

Reply:
I want to call it `abi` so I can use like this

```ts
encodeFunctionData({
  functionName: 'setAddr',
  abi,
  args: [node, '0x3a872f8FED4421E7d5BE5c98Ab5Ea0e0245169A0'],
})
```